magefiles/installation/install.go

```diff
@@ -218,11 +218,9 @@ func (i *InstallAppStudio) cloneInfraDeployments() error {
 		URL:           url,
 		ReferenceName: plumbing.ReferenceName(refName),
 		Progress:      os.Stdout,
+		RemoteName:    "upstream",
```

Comment:
not seeing this getting used anywhere, should we remove this?

Reply:
hmm not sure what you mean. the `upstream` remote name actually replaces the default `origin`, which is not used

Reply:
I totally missed the context and also I'm blind. makes sense now, thanks!